FlowCrypt/Functionality/Services/Local Private Key Services/KeyService.swift

```diff
@@ -71,12 +76,14 @@ final class KeyService: KeyServiceType {
             }
```

Comment:
The `// todo - should be refactored with https://github.com/FlowCrypt/flowcrypt-ios/issues/812` above is outdated - please help me remove